pkg/function/wait.go

```diff
@@ -69,15 +69,23 @@ func (*waitFunc) Name() string {
 }
 
 func (ktf *waitFunc) Exec(ctx context.Context, tp param.TemplateParams, args map[string]interface{}) (map[string]interface{}, error) {
+	rendered, err := param.RenderArgs(args, tp)
+	if err != nil {
+		return nil, err
+	}
+
 	var timeout string
-	var conditions WaitConditions
-	var err error
-	if err = Arg(args, WaitTimeoutArg, &timeout); err != nil {
+	if err := Arg(rendered, WaitTimeoutArg, &timeout); err != nil {
```

Comment:
Why do we need rendered arg to parse timeout? As per my understanding, it's always going to be const value.

Reply:
I prefer to retain the original behaviour where we used the rendered argument in `Arg()`, in this PR. I am curious - do we support Go template everywhere in the blueprint, or only specific fields?